app/components/homepage-components/ServicesCard.tsx

```diff
@@ -1,37 +1,57 @@
 "use client";
 
-import { useState } from "react";
-import Image from "next/image";
+import { motion } from "framer-motion";
 import { useTheme } from "next-themes";
+import { useState, useEffect } from "react";
+import Image from "next/image";
 
 import { ServiceProvidedType } from "@/types";
 interface ServicesCardProps {
   service: ServiceProvidedType;
+  delay?: number;
 }
 
-const ServicesCard = ({ service }: ServicesCardProps) => {
+const ServicesCard = ({ service, delay }: ServicesCardProps) => {
   const { theme } = useTheme();
   const [isMouseOver, setIsMouseOver] = useState<boolean>(false);
+  const [backgroundColor, setBackgroundColor] = useState("");
+  const [hoverBackgroundColour, setHoverBackgroundColour] = useState("");
 
-  let imageSrc;
+  const imageSrc = isMouseOver
+    ? theme === "light"
+      ? service.imageHoverLight
+      : service.imageHoverDark
+    : service.imageSrc;
 
-  if (isMouseOver) {
+  useEffect(() => {
     if (theme === "light") {
-      imageSrc = service.imageHoverLight;
+      setHoverBackgroundColour("#0252CD");
```

Comment:
Please update so you don't use hard-coded colors. These should all be in your tailwind config.